### PR TITLE
allow user to expand/collapse Notes List with keyboard

### LINKED ISSUE #330: cannot expand/collapse Notes List with keyboard
There's no way to expand/collapse a Notes List using the keyboard.

This will probably require making the expand/collapse icon a button.

Maybe the entire Notes List `<header>` could be a `<button>` instead. Hmm...

### PR DESCRIPTION
**Pull request by dwhieb**
**Related Issue:**

closes #330

**Description of Changes**

Allow the user to expand/collapse the Notes List using a keyboard.

### CODE REVIEW

**Review by vadekh — COMMENTED** [review]
This is not functional on my end. Every time I toggle the button the page seems to reload? Might be worth adding a test to the NotesList component to make sure this is working correctly.

**Comment by dwhieb** [comment]
@vadekh Just on Safari, or also for Chromium browsers?

**Comment by vadekh** [comment]
@dwhieb Unfortunately on both...

**Comment by dwhieb** [comment]
We debugged this and found that the issue is that the `<button>` is lacking the `type=button` attribute.

File: src/components/NotesList/NotesList.hbs [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <section class=notes-list>
   <header class=notes-list__header>
-    <svg aria-hidden=true class='notes-list__closed-icon notes-list__icon'><use href=#chevron-right></svg>
-    <svg aria-hidden=true class='notes-list__icon notes-list__opened-icon'><use href=#chevron-down></svg>
+    <button class='button notes-list__button notes-list__toggle-button transparent'>
```
Comment (vadekh) [inline src/components/NotesList/NotesList.hbs]:
```suggestion
    <button class='button notes-list__button notes-list__toggle-button transparent' type='button'>
```

**Review by vadekh — CHANGES_REQUESTED** [review]
I checked this change on my end and it does fix the issue.

**Review by vadekh — APPROVED** [review]

**Pull request merged**